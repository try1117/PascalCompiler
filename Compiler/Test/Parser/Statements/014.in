program test;
var a: integer;
begin
  if 'q' then a := 1;
  if 'qw' then a := 1;
end.
