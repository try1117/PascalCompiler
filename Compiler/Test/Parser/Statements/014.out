Lexical exception in position (5,11) - Incompatible types, expected Integer but found String
